src/deepsparse/server/server.py

```diff
@@ -275,26 +347,24 @@ def _predict_from_files(request: List[UploadFile]):
 
     routes_and_fns = []
     if integration == INTEGRATION_LOCAL:
-        route = endpoint_config.route or "/predict"
-        if not route.startswith("/"):
-            route = "/" + route
+        route = (
+            f"{endpoint_config.route}/infer"
+            if endpoint_config.route
+            else f"/v2/models/{endpoint_config.name}/infer"
+        )
+        route = clean_up_route(route)
 
         routes_and_fns.append((route, _predict))
         if hasattr(input_schema, "from_files"):
             routes_and_fns.append((route + "/from_files", _predict_from_files))
+
     elif integration == INTEGRATION_SAGEMAKER:
-        route = "/invocations"
+        route = "/invocations/infer"
```

Comment:
Same thought here about having a base class with children for each integration, to clean up the if statements

Reply:
same comment as above